test_pool/peripherals/test_d003.c

```diff
@@ -218,7 +218,7 @@ payload1()
           val_set_status(index, RESULT_PENDING(g_sbsa_level, TEST_NUM2));
           val_gic_install_isr(int_id, isr);
           uart_enable_txintr();
-          val_print_raw(g_print_level, "\n       Test Message                      ", 0);
+          val_print_raw(AVS_PRINT_DEBUG, "\n       Test Message                      ", 0);
```

Comment:
The reason to have g_print_level is to ensure that the "test message" is printed always and not fix it to a verbosity level. This is because we need the UART controller to generate an interrupt during the TX of that string. Making it DEBUG will not achieve that goal.